client-library-otel/wrangler.toml

```diff
@@ -0,0 +1,3 @@
+name = "hotel"
```

Comment:
note to self: needed?

Reply:
It's for the sample app. I use `npm run dev` and have a test app that immediately uses the lib that is located in the`src` folder